cobigen-templates/templates-devon4j/src/main/templates/crud_openapi_ionic_client_app/templates.xml

```diff
@@ -58,17 +58,17 @@
       <templateRef ref="app.module.ts"/>
       <templateRef ref="pageable.ts"/>
       <templateRef ref="sort.ts"/>
-      <templateRef ref="${variables.etoName#lower_case}-rest.service.ts"/>
-      <templateRef ref="${variables.etoName#lower_case}-list.page.ts"/>
-      <templateRef ref="${variables.etoName#lower_case}-list.page.html"/>
-      <templateRef ref="${variables.etoName#lower_case}-list.page.scss"/>
-      <templateRef ref="${variables.etoName#lower_case}-detail.page.ts"/>
-      <templateRef ref="${variables.etoName#lower_case}-detail.page.html"/>
-      <templateRef ref="${variables.etoName#lower_case}-detail.page.scss"/>
-      <templateRef ref="business-operator.service.ts"/>
-    </increment>
-    <increment name="ionic_theme" description="Ionic theme">
-      <templateRef ref="variables.scss"/>
+      <templateRef ref="${variables.etoName#lower_case}-rest.service.ts"/>
```

Comment:
Can you check the line endings characters? Git thinks there are changes in your files, but there shouldn't